locales/tr/packages/cashier.json

```diff
@@ -7,7 +7,7 @@
     "Extra confirmation is needed to process your payment. Please continue to the payment page by clicking on the button below.": "Ödemenizi işlemek için ek onay gereklidir. Lütfen aşağıdaki butona tıklayarak ödeme sayfasına devam ediniz.",
     "Full name": "Tam adı",
     "Go back": "Geri dön",
-    "Jane Doe": "Jane Doe",
+    "Jane Doe": "Ali Veli",
```

Comment:
@caouecs 

Ref: https://blog.plover.com/lang/Turkish-John-Doe.html

There were two alternatives. I preferred this. It is more useful because the number of letters is the same.

Reply:
@caouecs Sorry. I accidentally read it as "John" Doe. "Ali Veli" is a male name.

If you want to correct it, you can change it to: "Ayşe Fatma".